packages/nextjs/components/op/modals/VoteModal.tsx

```diff
@@ -78,6 +78,8 @@ const VoteModal: React.FC<IVoteModal> = ({
             onChange={e => handleAllocationChange(e.target.value)}
             className="border-none outline-none p-3 w-full  "
             value={allocation}
+            type="number"
+            step="0.01"
```

Comment:
This limitation of 0.01 increments is not going to work. OP tokens have 18 decimal places which means we should expect 0.000000000000000001 increments.

Reply:
True, agree with your points, but it will make UI hard to see, or the user has to enter either up to 18 decimals or use 1e18, we can also enable an option that the user can choose to enter the funds either in gwei, Wei form.